src/scss/_base.scss

```diff
@@ -25,3 +42,61 @@ main {
   margin: 0 auto;
   padding: 0 1em;
 }
+
+a {
+  text-decoration: none;
+  color: inherit;
+  font-weight: 700;
+  position: relative;
+  display: inline-block;
+}
+
+a:not(.footer *, .dark-bg *, .share *, .article-header *, .articles *, h2 *):hover {
+  opacity: 0.7;
+}
+
+a:not(.footer *, .button, .dark-bg *, .share *, .article-header *, .articles *, h2 *)::before {
+  content: "";
+  position: absolute;
+  top: 85%;
+  width: 100%;
+  left: 0;
+  height: 3px;
+  border-radius: 2px;
+  background: linear-gradient(to right, var(--color-accent-1), var(--color-accent-2));
+}
+
+h1 {
+  color: var(--color-text-darkbg);
+}
+
+p,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+  overflow-wrap: break-word;
+}
+
+ul > li + li,
+ol > li + li {
+  margin-top: 2rem;
+}
+
+::marker {
+  color: var(--color-red);
+}
+
+*:focus {
```

Comment:
Focus is an accessibility thing, make sure that whatever styles you write yourself overriding browser defaults are accessible and obvious.